src/api/handlers/PetFindManyHandler.ts

```diff
@@ -1,29 +1,29 @@
-import { Handler } from "aws-lambda";
+import { Handler, ProxyResult } from "aws-lambda";
+import { Database } from "../core/Database";
+import { ProxyResultBuilder } from "../core/ProxyResultBuilder";
 import { PetService } from "../services/PetService";
 
 class PetFindManyHandler {
-    private service: PetService;
+    private petService: PetService;
 
-    constructor() {
-        this.service = new PetService();
+    public async initializeDependencies(): Promise<void> {
+        const database = new Database();
+        try {
+            await database.connect();
+        } catch (e) {
+            await database.disconnect();
+            throw e;
```

Comment:
@bigLucas 
Adding `ProxyResultBuilder` I needed to instantiate the `Database` to get its `connection`. To solve this, I created the `Database` instance in the same function as the `PetService` instance.
In the PF project, I noticed that it has a `ServerlessDatabase` class and the handler extends that class.
I just made this commitment to show you how it is now.
Do you think I should create a new `ServerlessDatabase` class and follow PF inheritance structure or keep it as now?

Reply:
@bigLucas 
Reading M-0012 carefully, I found that I am doing more than necessary in this branch (M-0011).
Your idea was just to replace the current code inside `execute()` with the new one: `return new ProxyResultBuilder().status (200).body(response).build().`
And handle database stuffs in the next branch (M-0012).
Am I right?

Reply:
@Aristevao 
Yes, you're right!
You can reset the changes and just add the use of the builder.

Reply:
@Aristevao 
You also can add the builder in all files of the handler layer.